src/services/completions.ts

```diff
@@ -1776,6 +1776,18 @@ namespace ts.Completions {
             }
         }
 
+        // If the symbol is for a member of an object type and is the internal name of an ES
+        // symbol, it is not a valid entry. Internal names for ES symbols start with "__@"
+        if (symbol.flags & SymbolFlags.ClassMember) {

```

Comment:
I don't think internal names are specific to `ClassMember`s?

Reply:
ESSymbol named members are. `ClassMember` is a misnomer. It includes `Property`, `Method`, and `Accessor`, which are all of the ESSymbol-nameable members.